tests/cpp/operator/batchnorm_test.cc

```diff
@@ -1364,15 +1364,15 @@ TEST(BATCH_NORM, TestChannelAxisSaveAndLoad) {
 
 /*! \brief Insert the channel field `channelCount` into the shape at `channelAxis` position */
 static TShape MakeShape(const std::vector<index_t>& shape,
-                        unsigned int channelAxis,
+                        signed int channelAxis,
```

Comment:
Isn't int signed already?

Reply:
channelAxis has been referred to as signed int at multiple places in the test. Even this line had history where it was referred to as signed int. I guess to make it explicit. So I just stuck to that style. I can change them to int if that's better.

Reply:
Yes, it's better to keep the coding style consistent with the existing code base.